modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -11,10 +11,15 @@ error from another module like `FILE:FILE_NOT_FOUND`.
 
 Core errors must use the implicit namespace, while custom ones are forced to declare one.
 The custom error namespace must not be used elsewhere in your app, meaning that you cannot
-use `HTTP` as a namespace if you are using HTTP operations, for example. Once you
-declare a custom namespace by using it in a `raise-error`, you can use it in other
+use `HTTP` as a namespace if you are using HTTP operations.
+Once you declare a custom namespace by using it in a `raise-error`, you can use it in other
 `raise-error` components and custom types.
 
+Use the Raise Error component when you want your application to throw customized errors, besides the default automatic errors that are returned.
+As an example, if your application uses a HTTP connector and an error occurs, the connector throws its default automatic errors, for example `HTTP: NOT_FOUND` so you can not create a customized error using the same system namespace `HTTP`, for example `HTTP:MY_ERROR`. As a result, you have to create your own namespace for your customized error, for example `MYERROR:400HTTP`. Once set, and if there is an issue with the app connector, it will throw both the automatic default error and your own customized one.
```

Comment:
`if there is an issue with the app connector, it will throw both the automatic default error and your own customized one` I don't understand this. Only one error can occur at a time, you might catch it and change it, map it to another type once it occurs but only one will be generated.

Reply:
Removed paragraph.